src/popup/locales/en.json

```diff
@@ -75,6 +75,22 @@
       "btnText": "Create account",
       "btnSubtitle": "and add it to your wallet"
     },
+    "createMultisigAccount": {
+      "title": "Create new multisig vault",
+      "btnText": "Multisig account",
+      "btnSubtitle": "to manage funds with multiple accounts",
+      "signer": "Signer",
+      "signer-input-placeholder": "Signer's account address",
+      "add-signer": "Add Signer",
+      "add-signer-help-title": "Authorized signers",
+      "add-signer-help-msg": "Each multisig transaction needs to be confirmed by the signatures of a preset number of authorized accounts.<br /> <br />As the creator of this multisig vault you can determine the authorized signers and set the consensus required for a transaction confirmation.",
+      "consensus-required-desc": "Consensus required for tx confirmation:",
+      "consensus-required-content": "out of {0} signatures",
+      "consensus-required-help-title": "Consensus required",
+      "consensus-required-help-msg": "As the creator of this multisig vault you can determine the authorized signers and set the consensus needed for a transaction confirmation (how many signatures are required to confirm each transaction).",
+      "create-multisig-vault": "Create Vault",
+      "error-duplicating-signer": "Cannot add the same address as a signer twice"
```

Comment:
I suggest to use camelCase for the translation keys as we are moving towards this way of naming.